rq/job.py

```diff
@@ -1378,6 +1420,19 @@ def execute_failure_callback(self, death_penalty_class: Type[BaseDeathPenalty],
             logger.exception(f'Job {self.id}: error while executing failure callback')
             raise
 
+    def execute_stopped_callback(self, death_penalty_class: Type[BaseDeathPenalty]):
+        """Executes stopped_callback with possible timeout"""
+        if not self.stopped_callback:
+            return
+
+        logger.debug('Running stopped callbacks for %s', self.id)
+        try:
+            with death_penalty_class(self.stopped_callback_timeout, JobTimeoutException, job_id=self.id):
```

Comment:
Try using `SimpleWorker` for tests this execution gets tracked by `coverage.py`

Reply:
Yes, but that test have the order backwards. In `test_stopped_callback`. What we want is to have `job. execute_stopped_callback()` run in the foreground so its execution is detected by coverage.py. Hopefully this screenshot explains what I meant.

![CleanShot 2023-05-19 at 07 02 03](https://github.com/rq/rq/assets/188658/8e2469a4-3709-4a60-8906-85cf6de2a5a3)

Another simpler alternative is to call `job.execute_stopped_callback()` in the job creation test to validate that the function is properly persisted and can be correctly executed.